lib/themes/dosomething/paraneue_dosomething/templates/system/patterns/photo.tpl.php

```diff
@@ -17,7 +17,7 @@
   <?php if ($content->id && !$content->disable_reactions): ?>
     <ul class="form-actions -inline photo__actions">
       <li>
-        <a class="button -mini js-kudos-button <?php print dosomething_kudos_term_is_selected($content, 'heart') ? 'is-active' : '' ?>" data-kudo-id="<?php print array_pop($content->allowed_reactions); ?>">&#128150;</a>
+        <img class="js-kudos-button <?php print dosomething_kudos_term_is_selected($content, 'heart') ? 'is-active' : '' ?>" data-kudo-id="<?php print array_pop($content->allowed_reactions); ?>" src="https://files.slack.com/files-pri/T024GV2BW-F1EQSUM9P/download/gray-heart.svg" >
```

Comment:
PR at top says "not ready for review yet" for this reason, was easier to just use the slack url in my clipboard